doc/api/cli.md

```diff
@@ -280,6 +280,10 @@ effort to report stack traces relative to the original source file.
 Overriding `Error.prepareStackTrace` prevents `--enable-source-maps` from
 modifying the stack trace.
 
+Note, Accessing `Error.stack` with source maps enabled can be an expensive
```

Comment:
How about something like:

Note, enabling source maps can introduce latency to your application when `Error.stack` is accessed. If you access `Error.stack` in the hot path of your application, take into account the performance implications of `--enable-source-maps`.

Reply:
What exactly do we mean by `hot path` here? I am guessing it means some place which is called often?

How about
> Note, enabling source maps can introduce latency to your application when Error.stack is accessed. If you access Error.stack frequently in your application, take into account the performance implications of --enable-source-maps.

Reply:
Have made the above change ^. Let me know if it looks ok and if I understood it correct.